
syntax = "proto3";
package common;

option java_package = "io.grpc.personal";
option java_multiple_files = true;
option java_outer_classname = "PersonalProto";


// The PersonalInfo service definition.
service PersonalInfoService {

  rpc sendMessage (PersonalMessageRequest) returns (PersonalMessageReply) {}

}

// The request message containing the personal info.
message PersonalMessageRequest {
  string firstName = 1;
  string lastName = 2;

  // date of birth MMDDYYYY format
  string dob = 3;
  string email = 4;

  // phone number in XXX-XXX-XXXX format
  string phone = 5;
  string timeStamp = 6;

}

// The response message containing the status
message PersonalMessageReply {
  bool status = 1; // true means successful else false
}
